src/python/e2e-test-runner/e2e-test-runner.Dockerfile

```diff
@@ -0,0 +1,11 @@
+ARG TAG=latest
+
+#
+# e2e-tests
+################################################################################
+FROM grapl/graplctl:$TAG AS e2e-tests
+USER root
+RUN apt-get update && apt-get install -y --no-install-recommends unzip
+USER grapl
+COPY --chown=grapl ./dist/src.python.e2e-test-runner.src/lambda.zip lambda.zip
+RUN unzip lambda.zip
```

Comment:
We are unzipping the lambda because we can't actually invoke it as a lambda in localstack, instead we have to invoke it "by hand" in this container. Will fix the misnamed package.